src/featurecounts/config.vsh.yaml

```diff
@@ -0,0 +1,346 @@
+      arguments:
+        - name: --annotation
+          alternatives: ["-a"]
+          type: file
+          description: | 
+            Name of an annotation file. GTF/GFF format by default. See '--format' option for more format information.
+          required: true
+          example: annotation.gtf
+        - name: --input
+          alternatives: ["-i"]
+          type: file
+          multiple: true
+          multiple_sep: ';'
+          description: |
+            A list of SAM or BAM format files separated by semi-colon (;). They can be either name or location sorted. Location-sorted paired-end reads are automatically sorted by read names.
+          required: true
+          example: input_file1.bam
+    
+    - name: Outputs
+      arguments:
+        - name: --output
+          alternatives: ["-o"]
+          type: file
+          direction: output
+          description: | 
+            Name of output file including read counts. A separate file including summary statistics of counting results is also included in the output ('<output>.summary'). Both files are in tab delimited format.
+          must_exist: true
+          example: features.txt
+        - name: --output_summary
+          type: file
+          direction: output
+          description: | 
+            Summary statistics of counting results is also included in the output ('<output>.summary'). 
+          must_exist: true
+          example: features.txt.summary
+        - name: --output_junctions
+          type: file
+          direction: output
+          description: | 
+            Count of number of reads supporting each exon-exon junction. See '--junctions' for more information.
+          must_exist: false
+          example: features.txt.jcounts
+          must_exist: false
+        - name: --r_path
+          type: file
+          direction: output
+          description: | 
+            Directory to save the detailed assignment results. If unspecified, the directory where counting results are saved is used. See -R for more format information.
+          must_exist: false
+          example: detailed_results/
+          must_exist: false
+
+    - name: Annotation
+      arguments:
+        - name: --format
+          alternatives: ["-F"]
+          type: string
+          description: | 
+            Specify format of the provided annotation file. Acceptable formats include 'GTF' (or compatible GFF format) and 'SAF'. 'GTF' by default. 
+          choices: [GTF, GFF, SAF]
+          default: "GTF"
+          required: false
+        - name: --feature_type
+          alternatives: ["-t"]
+          type: string
+          description: | 
+            Specify feature type(s) in a GTF annotation. If multiple types are provided, they should be separated by ',' with no space in between. 'exon' by default. Rows in the annotation with a matched feature will be extracted and used for read mapping.
+          default: "exon"
+          required: false
```

Comment:
The actual implementation requires the multiple types to be provided as a string with types separated by a comma. But, if we use `multiple: true` it will be treated as an array and we would need additional processing steps.

Reply:
Okay, thanks for clearing that up.